xbmc/windowing/gbm/WinSystemGbm.h

```diff
@@ -20,6 +20,35 @@
 
 #include <gbm.h>
 
+#ifndef HAVE_HDR_OUTPUT_METADATA
```

Comment:
To keep platform-specific code out of headers, can this be moved into the cpp file? Then maybe forward-declare here.

```c++
struct hdr_metadata_infoframe;
struct hdr_output_metadata;
```

Reply:
The structs can also be used to a "helper header" like `xbmc/windowing/gbm/GbmTypes.h` but they're probably not used elsewhere.